src/main/java/org/example/exceptions/GlobalExceptionHandler.java

```diff
@@ -27,5 +27,11 @@ public ResponseEntity<Map<String, String>> handleNoDrivers(NoDriversException ex
         return ResponseEntity.status(HttpStatus.NOT_FOUND)
                 .body(Map.of("error", ex.getMessage()));
     }
+
+    @ExceptionHandler(InvalidRiderIDException.class)
```

Comment:
Global exception handler are used for handling generic exceptions being thrown in the controllers. Here, you can using it for specific exceptions which have certain context attached to them. 

We should throw context specific exceptions along with appropriate error messages in the relevant places only.